scripts/pxe.yaml

```diff
@@ -0,0 +1,49 @@
+#clear-linux-config
```

Comment:
Please add the same comment header and footer to this that is found in live-desktop.yaml as this will allow for a straight copy of our reference file to DevOps for publication.